data/presets/type/restriction/only_straight_on.json

```diff
@@ -11,5 +11,5 @@
         "key": "restriction",
         "value": "only_straight_on"
     },
-    "name": "Only Straight On"
+    "name": "No Turns"
```

Comment:
This is the most drastic change in the PR. As [this wiki page](https://wiki.openstreetmap.org/wiki/Relation:restriction#Road_signs) notes, signs in the United States literally say [<img src="https://upload.wikimedia.org/wikipedia/commons/2/2b/MUTCD_R3-3.svg" height="24" alt="No Turns" align="top">](https://commons.wikimedia.org/wiki/File:MUTCD_R3-3.svg), whereas “Only Straight On” is an odd-sounding construction for a couple different reasons.

Reply:
I would add that this wording makes a somewhat English-centric assumption. "No turns" does not imply "no U-turn" in languages where "turn" and "u-turn" are different words. As a similar example, "No left turn" (in right-side driving countries) does not forbid a "u-turn" by default. I feel like "No turns" sounds too much like "No left or right turn". Something like "straight only" removes this ambiguity and is easier to parse. Anyway, that's my impression from a non-US/non-native English speaker.

Reply:
This repository’s default locale is American English (`en-US`), so unfortunately there’s no other way to customize the wording for American English speakers other than to change the source string.

In principle, any language that would make a distinction between “turns” and U-turning can still use the appropriate terminology for this preset, including a literal translation of “Straight Only”. Unfortunately, I don’t think there’s currently a way for these preset files to supply comments to translators, so that context might be lost and translators might end up translating the string more literally.

In the meantime, “Straight Ahead Only” would be a reasonable compromise; thanks for the suggestion!